pkgs/top-level/all-packages.nix

```diff
@@ -2620,6 +2620,11 @@ with pkgs;
 
   _86Box = callPackage ../applications/emulators/86box { };
 
+  _86Box-with-roms = callPackage ../applications/emulators/86box {
```

Comment:
Use `.override` please, I asked that already.

Reply:
OK